CHANGELOG.md

```diff
@@ -4,6 +4,10 @@ CHANGELOG
 ## HEAD (Unreleased)
 _(none)_
 
+## 0.25.1 (2021-02-24)
```

Comment:
You should include the changelog item under `HEAD (Unreleased)`. When we cut a tagged version, we'll update the changelog to include all the unreleased items under a version heading.